### PR TITLE
build: quote more parts in PR validation report

### PR DESCRIPTION
**Pull request by jrudolph**
Currently on top of 4ce9afb which is known to fail with output in that segment.

### CODE REVIEW

File: project/ValidatePullRequest.scala [hunk 1]
```diff
@@ -393,7 +393,7 @@ object AggregatePRValidation extends AutoPlugin {
         def hasTests(result: Tests.Output): Boolean = result.events.exists(e => hasExecutedTests(e._2))
 
         def printTestResults(result: KeyValue[Tests.Output]): Unit = {
-          write(s"Test result for '${showKey(result.key)}'")
+          write(s"Test result for `${showKey(result.key)}``")
```
Comment (jrudolph) [inline project/ValidatePullRequest.scala:396]:
```suggestion
          write(s"Test result for `${showKey(result.key)}`")
```

**Comment by jrudolph** [comment]
See the failures above as an example. Now only the fixes are included here.

File: project/ValidatePullRequest.scala (hunk 1 above)
Comment (ennru) [reply project/ValidatePullRequest.scala:396]: This still needs to be applied, right?

Comment (jrudolph) [reply project/ValidatePullRequest.scala:396]: Ah, good catch, wonder what happened there...

**Pull request merged**